src/extensions/http_api/definitions.js

```diff
@@ -29,6 +29,14 @@ module.exports = ({ baseUrl = '' } = {}) => ({ Given, When, Then }) => {
         this.httpApiClient.setHeaders(Cast.object(this.state.populateObject(step.rowsHash())))
     })
 
+    /**
+     * Assign http headers,
```

Comment:
Maybe you should add a comment about the difference between "I set" & "I assign". Also remove the trailing comma or use a dot